src/mds/CInode.cc

```diff
@@ -3834,14 +3824,14 @@ void CInode::validate_disk_state(CInode::validated_data *results,
         shadow_in->fetch(get_internal_callback(INODE));
         return false;
       } else {
-        return fetch_dirfrag_rstats();
+	results->inode.passed = true;
```

Comment:
Why do you want to set inode.passed to true even though we haven't done anything with it? Since we haven't "checked" it, any callers shouldn't even be looking at the "passed" member.